packages/project/src/shared/schema.ts

```diff
@@ -11,6 +11,7 @@ export const Title = z
 
 export const ClonableSettings = z.object({
   isClonable: z.boolean().optional(),
+  isPublic: z.boolean().optional(),
```

Comment:
Aren't those exclusive?

Reply:
no, public is meant as in can be listed publicly on a directory, where clonable can be any project with even view link permission

Reply:
I'd replace isPublic with  visibility: private | public